spec/rails_helper.rb

```diff
@@ -65,6 +65,11 @@
     DatabaseCleaner.start
   end
 
+  config.before(:each) do
+    # Create root division
+    Division.create!(name: '-')
+  end
+
```

Comment:
What's the reasoning behind this change? Doesn't it mean an extra db create statement in cases where it's not needed?

Reply:
Yes, however it is a very fast Database query, and not having it results in some really weird Test failures that are hard to debug. So while this might add 1-2 seconds to the test suite run, I think it is worth it.